pkg/runtime/runtime.go

```diff
@@ -92,6 +92,10 @@ const (
 	middlewareComponent             ComponentCategory = "middleware"
 	defaultComponentInitTimeout                       = time.Second * 5
 	defaultGracefulShutdownDuration                   = time.Second * 5
+
+	// Pre-calculated limit to keep binding error message to 120 characters max.
+	grpcBodyLimit = 95
```

Comment:
What does those magic number mean?

Reply:
I wrote a new explanation for those numbers in [this commit](https://github.com/dapr/dapr/pull/3433/commits/61a20fdb4abcd1feb18bdfedcecef0d6a9c81859)

Reply:
If the log info changes, the number would make no sense, if use this way, we need to make info as const and do not change anymore, in fact I do not like this, maybe there is a way to auto detect?

WDYT? /cc @artursouza @yaron2

Reply:
I don't think there is a limit to keep this in 120. Just send the full error message.